src/basic_types.jl

```diff
@@ -74,13 +74,37 @@ struct BoundaryConditionDoNothing end
     return surface_flux(u_inner, u_inner, orientation_or_normal_direction, equations)
 end
 
+# This version can be called by hyperbolic solvers on logically Cartesian meshes
+@inline function (::BoundaryConditionDoNothing)(u_inner,
+                                                orientation_or_normal_direction,
+                                                direction::Integer, x, t,
+                                                surface_flux_functions::Tuple,
+                                                equations)
+    surface_flux_function, nonconservative_flux_function = surface_flux_functions
+    return surface_flux_function(u_inner, u_inner, orientation_or_normal_direction,
+                                 equations),
```

Comment:
```suggestion
    return surface_flux_function(u_inner, u_inner, 
                                 orientation_or_normal_direction, equations),
```

For nicer/unified formatting?